dll/win32/netcfgx/lang/tr-TR.rc

```diff
@@ -1,4 +1,4 @@
-// COPYRIGHT: 2014 Erdem Ersoy
+// COPYRIGHT: 2014 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
```

Comment:
For a fully qualified header you can use this suggestion (not a hard requirement though) :
```suggestion
/*
 * PROJECT:     ReactOS Configuration of network devices
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Turkish resource file
 * TRANSLATORS: Copytight 2014-2018 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
 *              Copytight 2021 Chan Chilung <eason066@gmail.com>
 */
```